src/ACF/Repeater.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace NanoSoup\Nemesis\ACF;
+
+/**
+ * Class Repeater
+ * @package NanoSoup\Nemesis\ACF
+ */
+class Repeater extends BaseFields
+{
+    /**
+     * Repeater init
+     *
+     * @param $args
+     * @return array
+     */
+    public function init($args)
+    {
+        $defaults = [
+            'button_label' => '',
+            'label' => '',
+            'prefix' => 'ss_acf',
+            'min' => '',
+            'max' => '',
+        ];
+
+        $settings = array_merge($defaults, $args['settings']);
+
+        return [
+            'key' => $this->generateUniquePrefix($settings['prefix'], $settings['label']) . 'field_repeater',
+            'label' => $settings['label'],
+            'name' => $this->generateName($settings['label']),
+            'type' => 'repeater',
+            'sub_fields' => $args['elements'],
```

Comment:
To do this like this you would then need it to call functions to add in the different fields to the group / repeater. How would this work, how would you actually add a group to a field group?

With flexible content it cycles through available functions to add them - so would suggest you look at passing through stuff like 'wysiwyg' etc. and then still making sure you can pass through labels etc.

Reply:
So I got it working by adding this to a `registerFieldGroup()` function in the same way as flexible content class worked (just called elements instead of layouts)

```
$this->group->init([
    'settings' => [
        'prefix' => $prefix,
        'label' => 'Group test'
    ],
    'elements' => [
        $this->text($prefix, 'Title'),
        $this->wysiwyg($prefix, 'Content')
    ]
])
```